.ci/scripts/util.sh

```diff
@@ -219,8 +219,11 @@ metrics_collection_end()
 
     # kill the background tasks (the pid number is stored in the file below)
     kill "$(cat "${stats_pid_file}")"
+    wait "$(cat "${stats_pid_file}")" 2> /dev/null || true
```

Comment:
What mechanism will end the processes that were previously being killed here?

Reply:
The `kill` on line 221 is still the mechanism that reaps the subproc, the `wait` was added so the stderr output about the killed process could be redirected to `/dev/null`. I lifted this approach from the answer to this thread: https://stackoverflow.com/questions/5719030/bash-silently-kill-background-function-process

Reply:
I see, I just made a mistake in looking at this as a line replacement. Thanks!